remote/rpc/clientV11.go

```diff
@@ -29,16 +29,16 @@ func NewClientV11(conn io.ReadWriteCloser) *RPCClientV11 {
 	return &RPCClientV11{NewClientV10(conn)}
 }
 
-func (p *RPCClientV11) ListServicesWithOptions(ctx context.Context, opts v11.ListServicesOptions) ([]v6.ControllerStatus, error) {
-	var resp ListServicesResponse
-	for _, svc := range opts.Services {
-		if err := requireServiceIDKinds(svc, supportedKindsV8); err != nil {
+func (p *RPCClientV11) ListWorkloadsWithOptions(ctx context.Context, opts v11.ListWorkloadsOptions) ([]v6.WorkloadStatus, error) {
+	var resp ListWorkloadsResponse
+	for _, svc := range opts.Workloads {
+		if err := requireWorkloadIDKinds(svc, supportedKindsV8); err != nil {
 			return resp.Result, remote.UnsupportedResourceKind(err)
 		}
 	}
 
-	err := p.client.Call("RPCServer.ListServicesWithOptions", opts, &resp)
-	listServicesRolloutStatus(resp.Result)
+	err := p.client.Call("RPCServer.ListWorkloadsWithOptions", opts, &resp)
```

Comment:
This depends on the server (which is the daemon) having the changed method name. Before we update client code -- used in Weave Cloud, for instance -- newer daemons will not respond correctly. More seriously, _after_ we update client code, older daemons will not respond correctly. In other words, this breaks backward compatibility, in a way that would be quite difficult to even paper over. I think it's probably better to let the existing versioned APIs remain as they are.